languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/prohibit.txt

```diff
@@ -6561,3 +6561,5 @@ Kreuzwehr
 Magenwehr
 Schädelwehr
 Zahnwehr
+Schutzschi
+Schutzschis/S
```

Comment:
Hier ist ein "s" zu viel.

Es reicht wenn man nur "Schutzschi/S" schreibt, dann werden "Schutzschi" und "Schutzis" ins Wörterbuch eingetragen

Reply:
Schutzschi**ss** kennt das LanguageTool auch. Exisitiert dieses Wort?

Reply:
LT akzeptiert viele Wörter, die aus zwei Nomen zusammengesetzt sind. Wenn die aber nicht nahe an einem anderen Wort liegen, ist das kein Problem. "Schutzschi" ist z.B. zwei Buchstaben von "Schutzschild" entfernt, da ist es schon sehr unwahrscheinlich, dass sich jemand so vertippt.

Reply:
unwahrscheinlich könnte es beim LT wirklich sein. Beim LO nicht.

Hier muss ich ehrlich sagen, dass ich die User der Erweiterung in anderen Bereichen wie Webbrowser nicht einschätzen kann. Ich nutze LT nur in LO und dort sind solche Erlaubnisse sehr nervig.

Wie sich solche Erlaubnisse z. B. im Webbrowser auswirken, kann ich nicht sagen.
Da kennt ihr euch besser aus.